shell/platform/darwin/ios/framework/Source/accessibility_bridge.mm

```diff
@@ -235,6 +235,10 @@ - (NSString*)routeName {
   return nil;
 }
 
+- (NSString *)accessibilityIdentifier {
+  return [NSString stringWithFormat:@"%d", self.node.id];
```

Comment:
Note: the node id is not stable. If you navigate away from a page and then back to the page the same elements there may have different nodes. Not sure if that matters...

Reply:
I added this only for testing(And add a comment on it). I started with testing based on label/value, but then I realized that we can't test pure container node (without a label/value) this way. If you can think of another way to verify a container node without a label/value, then we don't need this id.